Targets/CodeGenerator.targets

```diff
@@ -57,7 +57,7 @@
         {
         internal static class AssemblyCommitIds
         {
-        public const string LibGit2CommitSha = "$(libgit2_hash)"%3b
+        public const string LibGit2CommitSha = "e2e3f3e4cde7871f86eabacc6a639439b70a8fc7"%3b
```

Comment:
Hard-coding the hash here and in `GenerateNativeDllName.targets` since it refused to use the properties from the NuGet package. Not sure if it's a local thing, something not quite right with the way it's building, or something else, but it works fine if I hard-code the hash.

We won't want to merge these changes back into the upstream repo.